homeassistant/components/amcrest/camera.py

```diff
@@ -309,6 +310,11 @@ def model(self):
         """Return the camera model."""
         return self._model
 
+    @property
+    def unique_id(self) -> str:
```

Comment:
Typing is incorrect. It can return `None` in the statement below.
```suggestion
    def unique_id(self) -> str | None:
```

Reply:
Yes, I see some components are starting to specify the the type as nullable, which is more correct. Most components don't specify the type at all. Sorry - new to the project and I'm just trying to copy working code.
```
    306     def unique_id(self):
    168     def unique_id(self) -> str:
     32     def unique_id(self) -> str | None:
```